packages/hoppscotch-common/src/components/environments/Selector.vue

```diff
@@ -389,6 +392,33 @@ const teamEnvironmentList = useReadonlyStream(
   []
 )
 
+// Sort environments alphabetically by default
+const alphabeticallySortedPersonalEnvironments = computed(() => {
+  return [...myEnvironments.value]
+    .map((env, index) => ({
+      env,
+      index,
+    }))
+    .sort((a, b) =>
+      a.env.name
+        .toLocaleUpperCase()
+        .localeCompare(b.env.name.toLocaleUpperCase())
+    )
+})
```

Comment:
Can we abstract out the sorting logic as a helper function since it's consumed at multiple places?